addons/website/static/src/snippets/s_chart/000.js

```diff
@@ -8,6 +8,7 @@ const ChartWidget = publicWidget.Widget.extend({
     disabledInEditableMode: false,
     jsLibs: [
         '/web/static/lib/Chart/Chart.js',
+        '/web/static/lib/chartjs-adapter-luxon/chartjs-adapter-luxon.js',
```

Comment:
Something is making this snippet crash
![image](https://github.com/odoo/odoo/assets/30048408/f66fdc3d-927a-4a23-9dc8-578d8e70b0a7)

- Go to website app
- Enter edit mode
- Drag & Drop "Chart" snippet
- Change "type" to something else
- Crash